packages/note/src/react/__specs__/index.spec.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { render } from 'react-testing-library'
+
+import Note from '../index.js'
+
+describe('Note', () => {
+  it('exposes an Action Component', () => {
+    expect(Note).toHaveProperty('Action')
+  })
+
+  it('exposes a List Component', () => {
+    expect(Note).toHaveProperty('List')
+  })
+
+  it('renders the message', () => {
+    const { getByTestId } = render(
+      <Note message={<div data-testid="the-message" />} />
+    )
+
+    expect(getByTestId('the-message')).toBeInTheDocument()
+  })
+
+  it('renders metadata', () => {
+    const { container } = render(
+      <Note message={'message'} metadata={['meta1', 'meta2']} />
+    )
+
+    expect(container).toHaveTextContent('meta1')
+    expect(container).toHaveTextContent('meta2')
```

Comment:
I like these general render tests.  Probably because in the latest Tab commits I disabled storyshots, leaving it with essentially no tests.

Either we need to get hooks working correctly and reliably in storybook or move to tests more like this to compliment the storybook viz dev env.

Reply:
i haven't had issues with storybook yet...but i know that enzyme doesn't support hooks yet which could be an issue. hopefully we can just continue to use storyshots 🤞